src/silx/io/specfile/specfile.pyx

```diff
@@ -116,7 +116,7 @@ _logger = logging.getLogger(__name__)
 cimport cython
 from libc.stdlib cimport free
 
-cimport silx.io.specfile_wrapper as specfile_wrapper
+cimport specfile_wrapper
```

Comment:
Why did the cimport needed to be changed?

Reply:
It did not build. Anyway an absolute _C_import of a pxd file looks really odd to me. A relative import would have be better ...